doc/builders/trivial-builders.section.md

```diff
@@ -0,0 +1,53 @@
+# Trivial builders {#sec-trivial-builders}
```

Comment:
It also looks like you changed the anchor name here. The anchor names do not affect whether a document is chapter or section so this change only serves to break links.

Reply:
I think  `trivial-builders` was indeed a section and not a chapter by itself. Even because the recomendation of RyanTM in the original post:

https://github.com/NixOS/nixpkgs/issues/105763

Reply:
I think that recommendation is incorrect. It is a result of using a template for creating issues.

In the original XML file, you can clearly see that `chapter` is the root element and changing it to `section` will not work since the document is included in place where `chapter`s are expected (DocBook requires strict `book` → `chapter` → `section` hierarchy).